extension/src/gauge_agg.rs

```diff
@@ -1066,10 +1066,8 @@ mod tests {
         });
     }
 
-    // TODO https://github.com/timescale/timescaledb-toolkit/issues/362
-    // TODO why doesn't this catch the error under github actions?
-    // #[pg_test(error = "returned Datum was NULL")]
-    #[allow(dead_code)]
+    // TODO This is a terrible error message to show a human being.  We need to be a lot less cavalier about unwrap...
+    #[pg_test(error = "called `Option::unwrap()` on a `None` value")]
```

Comment:
All the other errors in here are decent quality!  I'll look at improving this error message in a follow-up :)